src/goto-instrument/accelerate/overflow_instrumenter.cpp

```diff
@@ -265,7 +271,7 @@ void overflow_instrumentert::overflow_expr(const exprt &expr, exprt &overflow)
   }
 }
 
-void overflow_instrumentert::fix_types(exprt &overflow)
+void overflow_instrumentert::fix_types(binary_exprt &overflow)
 {
   typet &t1=overflow.op0().type();
```

Comment:
Fix these?

Reply:
op0() and op1() on binary_exprt is ok; the fix is the type change.